src/components/NewsDetail.vue

```diff
@@ -29,5 +29,22 @@ export default {
   },
 }
 </script>
-<style>
+<style scoped lang="stylus">
+@import '~variables'
+
+.header-height
+  height 300px
+
+.news-detail-padding
+  padding 50px
+
+.news-detail-title-color
+  color $primary
+
+.news-detail-content-color
+  color grey
```

Comment:
color grey  -> can change to use color $variable ? like: color $neutral

Reply:
I only use setting color variable when it is used least twice times in app. 
Example : facebook color (it is used ast Register and Login page)

Reply:
Ok, I will use $neutral=grey

Reply:
$neutral is just an example, currently $neutral is #E0E1E2
the main point is to try our best to NOT introduce new colors, use primary/secondary/.... instead
unless it's really necessary to